include/SFML/Window/Vulkan.hpp

```diff
@@ -30,6 +30,7 @@
 ////////////////////////////////////////////////////////////
 #include <SFML/Window/Export.hpp>
 
+#include <cstddef>
```

Comment:
This include was added for `NULL` in 2503b002995e98e6beb12f890d9afb0c4c2c13ac but it is not needed on master as `nullptr` is used instead.